Sources/SwiftParser/TokenSpecSet.swift

```diff
@@ -455,6 +457,7 @@ enum DeclarationModifier: TokenSpecSet {
     case .static: return .keyword(.static)
     case .unowned: return TokenSpec(.unowned, recoveryPrecedence: .declKeyword)
     case .weak: return TokenSpec(.weak, recoveryPrecedence: .declKeyword)
+    case .resultDependsOnSelf: return TokenSpec(.resultDependsOnSelf, recoveryPrecedence: .declKeyword)
```

Comment:
Same here, you should check the experimental features.

Reply:
I don't think experimental check is needed here since the init handles the check.

Reply:
Oh, yes. Sorry, iI was a little overly eager here.